pkg/users/client.go

```diff
@@ -238,6 +238,44 @@ type ListAuthFactorsResponse struct {
 	ListMetadata common.ListMetadata `json:"listMetadata"`
 }
 
+type InviteObject struct {
+	Object         string `json:"object"`
+	ID             string `json:"id"`
+	Email          string `json:"email"`
+	State          string `json:"state"`
+	AcceptedAt     string `json:"accepted_at,omitempty"`
+	RevokedAt      string `json:"revoked_at,omitempty"`
+	Token          string `json:"token"`
+	OrganizationID string `json:"organization_id,omitempty"`
+	ExpiresAt      string `json:"expires_at"`
+	CreatedAt      string `json:"created_at"`
+	UpdatedAt      string `json:"updated_at"`
+}
+
+type CreateInvitationOpts struct {
```

Comment:
we call these "invite" / "invites" in our API so i think we should be consistent w/ that in the SDKs